snap/snapcraft.yaml

```diff
@@ -155,13 +155,23 @@ apps:
     plugs:
       - network
       - network-bind
+  vault-cli:
+    adapter: none
+    command: bin/vault
+    environment:
+      VAULT_CONFIG: "$SNAP_DATA/config/security-secret-store/vault-config.hcl"
+      VAULT_ADDR: "https://localhost:8200"
```

Comment:
if you wanted to be fancy, you could probably combine this with the yaml above with a yaml anchor. no preference from me, but if we need it for a third app I would prefer it.